services/kaze/controllers/application.go

```diff
@@ -228,6 +230,7 @@ func TransferApplicationOwnership(c *gin.Context) {
 
 // FetchMetrics retrieves the metrics of an application's container
 func FetchMetrics(c *gin.Context) {
+	c.Writer.Header().Set("Content-Type", "text/event-stream")
```

Comment:
this line is unnecessary as the response headers are automatically set during `c.SSEvent`